src/test/resources/log4j2-test.properties

```diff
@@ -1,11 +1,11 @@
-status = debug
+status = info
```

Comment:
Increased log level to remove any nonsense response-to-byte-array debug stuff from logs

Reply:
nice